.github/workflows/test.yml

```diff
@@ -15,13 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04]
-        php-version: [8.1]
-        experimental: [false]
-        include:
-          - os: ubuntu-20.04
-            php-version: 8.2
-            experimental: true
```

Comment:
in this file we can clean up `experimental` options as well

Reply:
done